lib/Ast_to_Mir.ml

```diff
@@ -294,7 +295,7 @@ let trans_prog filename
     let merge_maps maps =
       List.map ~f:Map.Poly.to_alist maps
       |> List.concat |> Map.Poly.of_alist_exn
-    and tvtables, stmts = List.map ~f:pull_tvdecls blocks |> List.unzip in
```

Comment:
I quite like this style of left-to-right composition. I also tried Googling the `|>` operator and it's easy to find. I hope it won't confuse anyone else though.